### ISSUE TITLE
OXMiq OP request

### ISSUE THREAD

**Issue by cmaryanTT** [tenstorrent/tt-metal#18123]
(aten/TOSA name)

P0 
aten.topk.default TOP_K
- [x] With issues below fixes, will; support K<=64
- [x] https://github.com/tenstorrent/tt-metal/issues/13235
- [x] https://github.com/tenstorrent/tt-metal/issues/11894

aten.sort.default SORT
- [x] Supported - https://github.com/tenstorrent/tt-metal/issues/18160

aten.view.dtype CAST
- [x] Supported

aten.convolution.default	CONV
- [ ] Supported, no clean C++ interface

aten.cumsum.default CUMSUM
- [x] Exists in TTNN as a Moreh OP

aten.mean.dim MEAN
- [x] Supported

aten.tanh.default TANH
- [x] Supported
- [x] Some minor PCC issues https://github.com/tenstorrent/tt-metal/issues/15132

P1
torch.nn.Upsample
https://docs.pytorch.org/docs/stable/generated/torch.nn.Upsample.html
- [ ] Not supported 

aten.abs.default ABS
- [x] Supported

aten.addmm.default ADDMM
- [ ] Not supported - can be lowered

aten.cumprod.default CUMPROD
- [x] Supported (currently in experimental, ttnn/operations/experimental/reduction/cumprod, needs to be moved out)

aten.eq.Scalar	EQUAL
- [x] Supported

aten.native_group_norm.default GROUP_NORM
- [x] Supported, align on detailed shapes

aten.pow.Tensor_Scalar	POW
- [x] Supported

P2
aten.exp.default EXP
- [x] Supported

aten.isin.Tensor_Tensor	INV_SIN
- [ ] Not supported

aten.native_layer_norm.default LAYER_NORM
- [x] Supported, align on detailed shapes

aten.lt.Tensor LESS
- [x] Supported

aten.lt.Scalar LESS_EQUAL
- [x] Supported

aten._scaled_dot_product_flash_attention.default ATTENTION
- [x] Supported, align on detailed shapes

dtype priorities:
bfloat16 float int32 int64 uint16 uint8 fp8
although whatever runs on bfloat16 should run on float at the same time and the float arithmetic should run with int32 concurrently

**Comment by cmaryanTT** [comment]
@ayerofieiev-tt @mbahnasTT - FYI

**Comment by aczajkowskiTT** [comment]
@cmaryanTT latest version works for K up to 3500, i.e. (1, 1, 32, 32768, 3, 3500) 
How does that affect this point above? [] With issues below fixes, will; support K<=64

**Comment by adifrancescoTT** [comment]
@cmaryanTT do you think we can close this once the TopK subissue is completed?

**Comment by cmaryanTT** [comment]
@adifrancescoTT - Not yet, the general functionality has been fixed, but there are still issues with the very long tensors requested here.

**Comment by fahmedchanTT** [comment]
@aczajkowskiTT or @cmaryanTT Would you please create/link open issues to the remaining items on the list here.

**Comment by aczajkowskiTT** [comment]
@fahmedchanTT Removed the int64 mentions as that will not be supported

**Comment by cmaryanTT** [comment]
@aczajkowskiTT - please review and see if this can be closed (even if some of this is tracked by other issues)

**Comment by aczajkowskiTT** [comment]
Yes, lets close this, there are individual tickets for the other outstanding items
